doc/maktaba.txt

```diff
@@ -1159,6 +1159,157 @@ maktaba#function#Sorted({list}, {func})            *maktaba#function#Sorted()*
   Returns a new list that is a sorted copy of {list}. {func} is used to
   determine the sort order, as in |sort()|.
 
+maktaba#internal#plugin#Enter({file})        *maktaba#internal#plugin#Enter()*
```

Comment:
Could we mark all the internal ones `@private` with public shims in `maktaba#plugin#` so there's no change to the public interface?

Reply:
I've actually done it in one of the previous versions, but it got lost somewhere in rebases.

Reply:
Done.

Reply:
Note, that I've marked Source as @dict Plugin, because it seems like a more appropriate comment.